apps/app/src/app/pages/(documentation)/documentation/(ui)/installation.page.ts

```diff
@@ -112,6 +112,53 @@ module.exports = {
 
 			<spartan-code class="mt-4" code="@import '@spartan-ng/brain/hlm-tailwind-preset.css';" />
 
+			<p class="${hlmP}">
+				If you are using VSCode and the
+				<a
+					hlmBtn
+					class="h-6 px-0.5 text-base underline"
+					size="sm"
+					href="https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss"
+					target="_blank"
+					variant="link"
+				>
+					Tailwind CSS IntelliSense
+				</a>
+				plugin, it is suggested to add the
+				<code class="${hlmCode}">hlm</code>
+				function to the tailwindCSS.classFunctions in either your workspace settings
+				<code class="${hlmCode}">.code-workspace</code>
+				file or VSCode's
+				<code class="${hlmCode}">settings.json</code>
+				to get all the Tailwind CSS IntelliSense plugin's features to work inside
+				<code class="${hlmCode}">hlm</code>
+				calls.
+			</p>
+
+			<spartan-code
+				code="
+// your-workspace.vscode-workspace
```

Comment:
Which file is it above you wrote .code-workspace and here its .vscode-workspace? You can check the recommendation from cva as well https://cva.style/docs/getting-started/installation#intellisense. 

I think adding it to the `settings.json` would be the best recommendation. It will either be the global `settings.json` or it may be in the `.vscode/settings.json`.